cli/chaosli/cmd/create.go

```diff
@@ -527,29 +527,39 @@ func getServices() []v1beta1.NetworkDisruptionServiceSpec {
 
 		hasPorts := confirmOption("Do you want to provide the ports affected by the disruption? (In case of no port defined, all ports of the service will be affected)", "")
 
-		ports := []v1beta1.NetworkDisruptionServicePortSpec{}
-
 		if hasPorts {
-			getServicePort := func() v1beta1.NetworkDisruptionServicePortSpec {
+			ports := []v1beta1.NetworkDisruptionServicePortSpec{}
+
+			getServicePort := func() *v1beta1.NetworkDisruptionServicePortSpec {
 				port := v1beta1.NetworkDisruptionServicePortSpec{}
 
+				fmt.Println(`We can identify a service port by name or by port value. At least one of those fields are required.`)
+
 				if confirmOption("Would you like to specify the name for this port?", "This field is optional and is used to find the right port to be affected in case the service has multiple ports") {
-					port.Name = getInput("Please enter the name of the port for this service (or ctrl+c to go back)", "")
+					port.Name = getInput("Please enter the name of the port for this service (or ctrl+c to go back)", "", survey.WithValidator(survey.Required))
 				}
 
-				if confirmOption("Would you like to specify the protocol for this port?", "This field is optional and is used to find the right port to be affected in case the service has multiple ports") {
-					port.Protocol, _ = selectInput("Please choose then (or ctrl+c to go back)", []string{"tcp", "udp", "sctp"}, "")
+				if confirmOption("Would you like to specify the port value for this port?", "This field is optional and is used to find the right port to be affected in case the service has multiple ports") {
+					port.Port, _ = strconv.Atoi(getInput("Please enter the name of the port for this service (or ctrl+c to go back)", "", survey.WithValidator(integerValidator), survey.WithValidator(survey.Required)))
 				}
 
-				port.Port, _ = strconv.Atoi(getInput("What port would you like to target?", "", survey.WithValidator(integerValidator), survey.WithValidator(survey.Required)))
+				if port.Name == "" && port.Port == 0 {
+					fmt.Println(`No port name or port value was specified.`)
```

Comment:
```suggestion
					fmt.Println(`No port name or port value was specified, all Kubernetes service ports will be impacted`)
```

Reply:
Here the user didn't provide any port or value where they should have. To my understanding this case would only happen if the user made a mistake while responding to the questions, but the user can still add another service port after this, so not all ports would be impacted here. 
We could however interpret this lack of response to signify that they don't want any service ports and the message would be relevant here. WDYT?

Reply:
🤔 I would continue your previous reasoning about being strict.
If it chooses to provide a port or name and don't, he should receive an error message and be re-asked (or able to exit).

Maybe we could clarify in this message how to get back to not select any port.

```suggestion
					fmt.Println(`No port name or port value provided, please provide a port/name or hit CTRL^C to skip this filtering and impact all ports.`)
```

Reply:
This was my error message but yes it's probably not enough. Thanks!